cpp/src/gandiva/tests/test_util.h

```diff
@@ -101,4 +103,16 @@ static inline std::shared_ptr<Configuration> TestConfiguration() {
   return builder.DefaultConfiguration();
 }
 
+#ifndef GANDIVA_EXTENSION_TEST_DIR
+#define GANDIVA_EXTENSION_TEST_DIR "."
+#endif
+
+static inline Status LoadTestLLVMIR() {
```

Comment:
Fixed. I added a `test_util.cc` file, and since previously all gandiva projector tests are added via `add-gandiva-test` function and no additional test common files is ever used, I changed the gandiva/tests/CMakeLists.txt as well.